Packs/MandiantAdvantageThreatIntelligence/Integrations/MandiantAdvantageThreatIntelligence/MandiantAdvantageThreatIntelligence.py

```diff
@@ -1112,7 +1112,7 @@ def fetch_indicators(client: MandiantClient, args: dict = None) -> tuple[List, d
     if not args:
         args = {}
 
-    limit = args.get("limit", client.limit)
+    limit = int(args.get("limit", client.limit))
```

Comment:
Please notice you can also use a function from commonServerPython called arg_to_number that deals with this as well and covers other cases as well.